convert_hf_to_gguf.py

```diff
@@ -5825,9 +5825,11 @@ class Gemma3Model(TextModel):
     norm_shift = 1.0  # Gemma3RMSNorm adds 1.0 to the norm value
 
     def set_vocab(self):
-        self._set_vocab_sentencepiece()
-
-        self.gguf_writer.add_add_space_prefix(False)
+        if (self.dir_model / "tokenizer.json").is_file():
+            self._set_vocab_gpt2()
+        else:
+            self._set_vocab_sentencepiece()
+            self.gguf_writer.add_add_space_prefix(False)
```

Comment:
```suggestion
        if (self.dir_model / "tokenizer.model").is_file():
            self._set_vocab_sentencepiece()
            self.gguf_writer.add_add_space_prefix(False)
        else:
            self._set_vocab_gpt2()
```
I see that Gemma3 repo has both, while it might work just fine, let's not change conversion for those.

Reply:
Ah, I didn't realize Gemma3 had a tokenizer.json.  Makes sense.